x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/event_filters_flyout.tsx

```diff
@@ -72,6 +73,10 @@ export const EventFiltersFlyout: React.FC<EventFiltersFlyoutProps> = memo(
       getInitialExceptionFromEvent(data)
     );
 
+    const [confirmModalLabels, setConfirmModalLabels] = useState<ArtifactConfirmModalLabelProps>();
```

Comment:
Should the type for the `useState()` here be `ArtifactConfirmModalLabelProps | undefined` since you are not setting a default value?